mqtt-streaming/src/test/scala/docs/scaladsl/MqttSessionSpec.scala

```diff
@@ -1037,14 +1037,14 @@ class MqttSessionSpec
               .clientSessionFlow(session)
               .join(pipeToServer)
           )
-          .toMat(Sink.ignore)(Keep.both)
+          .toMat(Sink.headOption)(Keep.both)
           .run()
 
       val connect = Connect("some-client-id", ConnectFlags.None)
 
       client.offer(Command(connect))
 
-      result.failed.futureValue shouldBe a[WatchedActorTerminatedException]
```

Comment:
This exhibits the core of my change here - this termination exception isn't something that the caller should be concerned with. It is internal to the session.